### ISSUE TITLE
Update My Samples page content for all sites (January release) (Biospecimen)

### ISSUE THREAD

**Issue by erincschwartz** [episphere/connect#1153]
We are incorporating guidance around waiting to donate samples for Connect following platelet donation as well as updating current guidance around the wait time associated with plasma donation. The My Samples pages for all sites needs to be updated. Tracked versions are linked below: 

KPCO: https://nih.app.box.com/file/1693285295439
KPGA: https://nih.app.box.com/file/1693248754879
KPHI: https://nih.app.box.com/file/1693267375005
KPNW: https://nih.app.box.com/file/1693245657249
BSWH: https://nih.app.box.com/file/1694253822276
HFH: https://nih.app.box.com/file/1731008430385
Marshfield: https://nih.app.box.com/file/1694261430922
HP: https://nih.app.box.com/file/1694270330251
UChicago: https://nih.app.box.com/file/1694262966027
Sanford: https://nih.app.box.com/file/1730947499980

**Comment by sonyekere** [comment]
The aim is to have tracked changes be finalized and ready for DevOps to incorporate the changes by December 15th. @jhflorey , This work will occur in the Jan release.

**Comment by sonyekere** [comment]
@jhflorey , we may assign this to you in the Jan release.

**Comment by brotzmanmj** [comment]
@erincschwartz Hi Erin, Can you confirm if the links above final content now from all the sites?

**Comment by erincschwartz** [comment]
Yes, the links above are the final content from all the sites.  

Spanish content updates for all sites, except HFH are highlighted below: 
![Image](https://github.com/user-attachments/assets/1700ca60-e030-4c31-899a-3179e17e0f02)

HFH content is with NORC for translation. @depietrodeanna, please post the HFH Spanish updates in this issue when available.

**Comment by erincschwartz** [comment]
HP's My Samples content linked above is now final, Spanish translations are forthcoming.

**Comment by jhflorey** [comment]
> @brotzmanmj, the links above are the final content from all the sites, except HP (waiting on them to provide the updates for Park Nicollet location, will update here when that is received)
> 
> **Spanish content updates for all sites, except HFH are highlighted below:** ![Image](https://private-user-images.githubusercontent.com/157617486/399790078-1700ca60-e030-4c31-899a-3179e17e0f02.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0NDMwNzUsIm5iZiI6MTczNjQ0Mjc3NSwicGF0aCI6Ii8xNTc2MTc0ODYvMzk5NzkwMDc4LTE3MDBjYTYwLWUwMzAtNGMzMS04OTlhLTMxNzllMTdlMGYwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxNzEyNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmI1ZTliMTBkMWM5M2EyZDQ0NGNhODNmYWQ5ODZkYTYwMmZhMGMyNDE0MzI3ZGJiMmQ1MWQzODVlOTc5YTU2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LVgnj8wItrShlmCfKsHHG5MFMJQuvPpha0nstE2URZs)
> 
> **And Chicago has 2 additional edits:** Change "del estudio" to "...del laboratorio" where highlighted below: ![Image](https://private-user-images.githubusercontent.com/157617486/399790825-c7f538fb-57b5-4e7d-a7a0-75a1ed28cb85.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0NDMwNzUsIm5iZiI6MTczNjQ0Mjc3NSwicGF0aCI6Ii8xNTc2MTc0ODYvMzk5NzkwODI1LWM3ZjUzOGZiLTU3YjUtNGU3ZC1hN2EwLTc1YTFlZDI4Y2I4NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxNzEyNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NDc3NDAyMThiOTA1ZTk4M2Q2MDNiYWZjOTNmY2JkMjM5Mjc0MmY5NjQ1ZTM3NGQwMjg5YjFiYTM4OTg1MzczJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wnDeSjNruidjE5KEX_he2dqo4j65S_wqU88jJXxiNTo)
> 
> **and**
> 
> ![Image](https://private-user-images.githubusercontent.com/157617486/399790946-7a4b7b9e-2d8f-4b7a-87b3-b54e6f48697c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0NDMwNzUsIm5iZiI6MTczNjQ0Mjc3NSwicGF0aCI6Ii8xNTc2MTc0ODYvMzk5NzkwOTQ2LTdhNGI3YjllLTJkOGYtNGI3YS04N2IzLWI1NGU2ZjQ4Njk3Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxNzEyNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDMyNTc3Mzk5NDU2YWFlNDZlMjRmMzczYzAxNDQ2MzhjNzYwMTQ1YTBmMjg3ZTNiM2IyZjc2ODA1ZTFmYTY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.E53tPqtuz2vwxEMl1NceFxshWfl6FeN4cOuL2ysI3Gk)
> 
> HFH content is with NORC for translation. @depietrodeanna, please post the HFH Spanish updates in this issue when available.

@erincschwartz  Could you please paste the Spanish text into new comment. I can not copy the text from your images.

**Comment by erincschwartz** [comment]
@jhflorey, the Spanish translations are below. I am unable to highlight in Git Hub, so I italicized the text that needs to be edited for your awareness (it does not need to be italicized on the actual My Samples pages). 

For all sites, except for HFH and HP (they recently added another clinical collection location, translation is forthcoming)

**¿Cuándo debería donar mis muestras de sangre y de orina?**

Cuando llegue el momento de donar sus muestras, le escribiremos por correo electrónico. Es importante que done sus muestras lo antes posible después de recibir el mensaje. 
**Nota:** Si hace poco tiempo le han hecho una transfusión de sangre o ha donado sangre, espere **al menos ocho semanas** desde su donación o transfusión antes de donar muestras para Connect. Si hace poco tiempo ha donado plasma, espere **al menos _cuatro semanas_** desde su donación de plasma antes de donar muestras para Connect. _Si hace poco tiempo ha donado plaquetas, espere **al menos una semana** desde su última donación de plaquetas antes de donar muestras para Connect._ Si van a hacerle una colonoscopia dentro de poco tiempo, no done muestras para Connect el mismo día de la colonoscopia.

Working on getting the translation needed for the 2 small additional edits for UChicago. Either @depietrodeanna or I will add these ASAP.

**Comment by depietrodeanna** [comment]
@jhflorey The additional edits for UC are to change **one word** in each paragraph below:

Change "del estudio" to "...del laboratorio." I bolded the changed word below:

Under heading **Instrucciones de estacionamiento** for UChicago Medicine River East:
Una vez dentro del estacionamiento, siga las señales hacia LL3 (Fall) hasta llegar al estacionamiento designado para pacientes de UChicago Medicine. Validaremos su pase de estacionamiento desde el estacionamiento. Muéstrele su ticket de estacionamiento sin asistencia al personal del **laboratorio**. No podemos validar el estacionamiento en la calle.

AND under the same heading for UChicago Medicine South Loop, in Downtown Chicago, 
La clínica está en el nivel P1, al lado del DSW. Las entradas se encuentran junto a Canal St., justo al lado de Panera Bread y en el nivel P1 del estacionamiento. Validaremos su pase de estacionamiento desde el estacionamiento. Muéstrele su ticket de estacionamiento sin asistencia al personal del **laboratorio**. No podemos validar el estacionamiento en la calle.

Please let me know if you need anything else for UC

**Comment by depietrodeanna** [comment]
@jhflorey the highlighted text here is where there are changes in the Spanish version: https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1745762451809&sharedAccessCode=

Please note that the updated version completely removes the list of locations under the subheadings "Opción 1: Laboratorios de investigación de Connect" and "Opción 2: Centros de Servicios de Laboratorio de Henry Ford Health"

**Comment by jhflorey** [comment]
@erincschwartz @depietrodeanna could you confirm this text is `for` or `For`?
<img width="579" alt="image" src="https://github.com/user-attachments/assets/b27dae19-43a0-4f8a-8cf8-2d0b4473d791" />

**Comment by erincschwartz** [comment]
Please use "for"

**Comment by jhflorey** [comment]
@erincschwartz got it. CC @we-ai

**Comment by jhflorey** [comment]
@erincschwartz @depietrodeanna @rohanjay10 My code change has been merged into dev. We can plan to test it.

**Comment by erincschwartz** [comment]
Thank you @jhflorey, I have reached out to @depietrodeanna to come up with a plan to review all of these My Samples pages in Dev.

**Comment by depietrodeanna** [comment]
@jhflorey HFH My Samples, there are a few updates needed in Dev for both English and Spanish. I made comments in the documents for each to show you where these places are: 
English:https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1731008430385&sharedAccessCode=
Spanish: https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1745762451809&sharedAccessCode=

**Comment by depietrodeanna** [comment]
@jhflorey 
Dev tested and here are the outcomes:

MF and SF look good

HP- a couple of minor edits to make, indicated by comments on the My Samples pages for English (https://nih.app.box.com/file/1694270330251) and Spanish (https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1746923787810&sharedAccessCode=)

Chicago has a minor revision or two needed, noted in the My Samples pages:
English-https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1694262966027&sharedAccessCode=
Spanish- see above comment with screenshots. The two places where a word switch is needed are under "South Loop" and "River East" locations. The word "estudio" needs to be added to replace "laboratorio"

**Comment by erincschwartz** [comment]
Jaime also added a new clinical collection location within the tracked document linked above. @depietrodeanna, can you add Crown Point to the Spanish version quick?

**Comment by erincschwartz** [comment]
@jhflorey KPCO, KPGA, KPHI and KPNW look good.

**Comment by depietrodeanna** [comment]
@jhflorey 

For Chicago My Samples in **Spanish**, please add the following for the new Crown Point location under the Kenwood location:

<img width="459" alt="Image" src="https://github.com/user-attachments/assets/e131c6e4-a4fe-42c9-82a9-de210ad1788f" />

**UChicago Medicine Crown Point**

Dirección e indicaciones

10855 Virginia St 
Crown Point, IN 46307 

The **English** version of the My Samples page also needs this location added. It is a tracked change in the document and I added a comment to direct you to it. https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1694262966027&sharedAccessCode=

**Comment by jhflorey** [comment]
@erincschwartz @depietrodeanna My changes are ready in dev. Please test again.

**Comment by erincschwartz** [comment]
@jhflorey I just reviewed the BSWH page and there are two edits, noted in comments of this document: BSWH: https://nih.app.box.com/file/1694253822276

**Comment by depietrodeanna** [comment]
> https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1731008430385&sharedAccessCode=

@jhflorey for HFH English page, Please capitalize the "S" in email ConnectStudy@hfhs.org and close parentheses after "...call you back." This is under the last section "Questions? Contact the Connect Study Team at Henry Ford Health"

<img width="995" alt="Image" src="https://github.com/user-attachments/assets/56460721-b0ae-41b9-90df-e37cd8eea60c" />

**Comment by depietrodeanna** [comment]
@jhflorey tested and last note, Chicago looks good. 

HP has one edit needed in the **English version only**. Please remove the hyphen in "check-in." See below. It should be "check in"

<img width="451" alt="Image" src="https://github.com/user-attachments/assets/6aaa96ad-fd22-47a6-a6cf-c833d75c01f0" />

**Comment by jhflorey** [comment]
> [@jhflorey](https://github.com/jhflorey) I just reviewed the BSWH page and there are two edits, noted in comments of this document: BSWH: https://nih.app.box.com/file/1694253822276

@erincschwartz please provide Spanish version for this text.

`Call 1-877-505-0253 (8:00 a.m. - 10:00 p.m. CT on weekdays and 9:00 a.m. - 6:00 p.m. CT on weekends). `

**Comment by depietrodeanna** [comment]
@jhflorey comments added in this document to indicate changes needed in Stage My Samples page for Spanish https://nih.app.box.com/file/1745762451809?s=7u67ji2b23mw2bn2hut96nhx05to77i2

**Comment by robertsamm** [comment]
@jhflorey for the stage version of the Marshfield MySamples page, the following changes are needed for the hours listed for the Minocqua Center and Weston Center. Changes are needed both in English and Spanish.

![Image](https://github.com/user-attachments/assets/e3bcc4a1-c348-4f3e-a95e-0e83e03250ec)

- Minocqua English: Tuesdays: 8:00 a.m. to 4:00 p.m. Wednesdays: 8:00 a.m. to 12:00 p.m. 
- Minocqua Spanish: Martes: 8:00 a.m. to 4:00 p.m. Miércoles 8:00 a.m. to 12:00 p.m.
- Weston English: Tuesday – Thursday: 7:00 a.m. to 1:00 p.m.
- Weston Spanish: De martes a jueves: 7:00 a.m. to 1:00 p.m.

**Comment by erincschwartz** [comment]
@jhflorey There are inconsistencies in HP My Samples bolding:

<img width="893" alt="Image" src="https://github.com/user-attachments/assets/cc3decbd-7e15-4977-bdff-b107a2a99c7e" />

Please just bold the **una semana** to be consistent.

**Comment by erincschwartz** [comment]
@jhflorey there are many inconsistencies in Stage from what is in the tracked HP Spanish document: 
https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1746923787810&sharedAccessCode=

I have added comments where the PWA needs to be updated

**Comment by robertsamm** [comment]
@jhflorey BSWH updates look good in dev for both English and Spanish

**Comment by jhflorey** [comment]
@robertsamm `Minocqua Spanish: Martes: 8:00 a.m. to 4:00 p.m. Miércoles 8:00 a.m. to 12:00 p.m.` Should this be 'de' instead of 'to'?

**Comment by erincschwartz** [comment]
it should be 'a' ----"8:00am a 4:00pm'

**Comment by robertsamm** [comment]
Thanks for the catch @jhflorey and @erincschwartz. And Weston Spanish should also change 'to' to 'a': De martes a jueves: 7:00 a.m. a 1:00 p.m.

**Comment by robertsamm** [comment]
@jhflorey UC looks good in dev

**Comment by jhflorey** [comment]
> [@jhflorey](https://github.com/jhflorey) for the stage version of the Marshfield MySamples page, the following changes are needed for the hours listed for the Minocqua Center and Weston Center. Changes are needed both in English and Spanish.
> 
> ![Image](https://github.com/user-attachments/assets/e3bcc4a1-c348-4f3e-a95e-0e83e03250ec)
> 
> * Minocqua English: Tuesdays: 8:00 a.m. to 4:00 p.m. Wednesdays: 8:00 a.m. to 12:00 p.m.
> * Minocqua Spanish: Martes: 8:00 a.m. to 4:00 p.m. Miércoles 8:00 a.m. to 12:00 p.m.
> * Weston English: Tuesday – Thursday: 7:00 a.m. to 1:00 p.m.
> * Weston Spanish: De martes a jueves: 7:00 a.m. to 1:00 p.m.

@robertsamm it's ready in dev. Please check again.

**Comment by jhflorey** [comment]
> [@jhflorey](https://github.com/jhflorey) HFH updates in Spanish look good in Dev. Still need to update the English bolding as I noted above for the word "participating"
> 
> ![Image](https://github.com/user-attachments/assets/1ae7d0da-903d-4495-9827-8aa2ce4e7a6b)

@depietrodeanna It's ready in dev. Please check again.

**Comment by jhflorey** [comment]
> [@jhflorey](https://github.com/jhflorey) there are many inconsistencies in Stage from what is in the tracked HP Spanish document: https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1746923787810&sharedAccessCode=
> 
> I have added comments where the PWA needs to be updated

@erincschwartz It's ready in dev. Please check again.

**Comment by robertsamm** [comment]
@jhflorey MF and HFH also look good in dev. Ready for stage.

**Comment by robertsamm** [comment]
Latest status of all site MySamples pages testing to bring it all under one comment:
- HP – looks good in dev, need to push to stage and retest
- MF – looks good in dev, need to push to stage and retest
- HFH – looks good in dev, need to push to stage and retest
- UC – looks good in dev, need to push to stage and retest
- BSWH – looks good in dev, need to push to stage and retest
- SH – ready for prod
- KPCO – ready for prod
- KPNW – ready for prod
- KPGA – ready for prod
- KPHI – ready for prod 

@sonyekere is the plan to still get these pushed to stage and tested this evening with a prod release tonight? Or do you think the stage testing will linger into tomorrow morning to finish up?

**Comment by depietrodeanna** [comment]
> > [@jhflorey](https://github.com/jhflorey) HFH updates in Spanish look good in Dev. Still need to update the English bolding as I noted above for the word "participating"
> > ![Image](https://github.com/user-attachments/assets/1ae7d0da-903d-4495-9827-8aa2ce4e7a6b)
> 
> [@depietrodeanna](https://github.com/depietrodeanna) It's ready in dev. Please check again.

This looks good now @jhflorey

**Comment by jhflorey** [comment]
> Latest status of all site MySamples pages testing to bring it all under one comment:
> 
> * HP – looks good in dev, need to push to stage and retest
> * MF – looks good in dev, need to push to stage and retest
> * HFH – looks good in dev, need to push to stage and retest
> * UC – looks good in dev, need to push to stage and retest
> * BSWH – looks good in dev, need to push to stage and retest
> * SH – ready for prod
> * KPCO – ready for prod
> * KPNW – ready for prod
> * KPGA – ready for prod
> * KPHI – ready for prod
> 
> [@sonyekere](https://github.com/sonyekere) is the plan to still get these pushed to stage and tested this evening with a prod release tonight? Or do you think the stage testing will linger into tomorrow morning to finish up?

The remaining 5 sites are ready in stage.

**Comment by robertsamm** [comment]
@sonyekere @jhflorey @anthonypetersen MySamples pages all look good in Stage. No addtl changes needed.

**Comment by anthonypetersen** [comment]
Everything that was in dev yesterday has been pushed to prod

**Comment by robertsamm** [comment]
Thanks for confirming Tony!
